crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -1084,11 +965,37 @@ impl actor::HcP2p for HolochainP2pActor {
 
     fn get(
         &self,
-        _dna_hash: DnaHash,
-        _dht_hash: holo_hash::AnyDhtHash,
-        _options: actor::GetOptions,
+        dna_hash: DnaHash,
+        dht_hash: holo_hash::AnyDhtHash,
+        options: actor::GetOptions,
     ) -> BoxFut<'_, HolochainP2pResult<Vec<WireOps>>> {
-        Box::pin(async move { todo!() })
+        Box::pin(async move {
+            let kitsune = self.kitsune()?;
+            let space_id = dna_hash.to_k2_space();
+            let space = kitsune.space(space_id.clone()).await?;
+            let loc = dht_hash.get_loc();
+
+            let (to_agent, to_url) = self.get_peer_for_loc("get", &space, loc).await?;
```

Comment:
I *think* this is essentially what current holochain (like v 0.4) is doing. That is, just returning an error from the first peer it is able to make the get request to.

But we should consider if this is what we *want* it to be doing. Because we could loop here for the duration of the timeout trying additional peers or something.